pkpdapp/pkpdapp/settings.py

```diff
@@ -164,16 +164,62 @@
 # Forever cachable files and compression support
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
 
+PLOTLY_DASH = {
+
+    # Route used for the message pipe websocket connection
+    "ws_route": "dpd/ws/channel",
+
+    # Route used for direct http insertion of pipe messages
+    "http_route": "dpd/views",
+
+    # Flag controlling existince of http poke endpoint
+    "http_poke_enabled": True,
+
+    # Insert data for the demo when migrating
+    "insert_demo_migrations": False,
+
+    # Timeout for caching of initial arguments in seconds
+    "cache_timeout_initial_arguments": 60,
+
+    # Name of view wrapping function
+    "view_decorator": None,
+
+    # Flag to control location of initial argument storage
+    "cache_arguments": True,
+
+    # Flag controlling local serving of assets
+    "serve_locally": False,
+}
+
+# Staticfiles finders for locating dash app assets and related files
+
+STATICFILES_FINDERS = [
+
+    'django.contrib.staticfiles.finders.FileSystemFinder',
```

Comment:
Can we make these alphabetical?

Reply:
I'm easy about it. You could group by functionality (with comments to label groups of functions) and do alphabetical within them but really not an important thing!